src/main/resources/application-jpa.yml

```
@@ -16,4 +17,10 @@ spring:
       hibernate:
         format_sql: true
         dialect: org.hibernate.dialect.MySQL8Dialect
-    open-in-view: false
\ No newline at end of file
+    open-in-view: false
+
+  data:
+    redis:
+      host: ${REDIS_HOST}
+      password: ${REDIS_PASSWORD}
+      port: ${REDIS_PORT}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)